src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2389,6 +2458,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
```

Comment:
>Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation) [](start = 8, length = 79)

BoundLateBoundArgumentSupportingAssignmentWithCapture looks like a lower node, no need to modify it.